.github/workflows/ccpp.yml

```diff
@@ -10,6 +10,7 @@ on:
 
 jobs:
   build:
+    if: "!(contains(github.event.head_commit.message, 'skip') && contains(github.event.head_commit.message, 'ci'))"
```

Comment:
Is it possible to make the check stricter?
A commit message like `Should not skip build while updating scientific publications` (note the not in front of skip, and the ci in scientific) would trigger an unexpected ci skip.

Reply:
Agree - `[skip ci]` and `[ci skip]` seem to be the standard strings.

Reply:
It is  now skipped when the commit message contains` [skip ci] `or `[ci skip]`,